hr_contract_amendment/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=====================
+hr contract amendment
+=====================
+
+This module extends the functionality of HR contract to keep history of change
+and to allow you to saw when and who made contract change
+
+For usability field 'notes' is not save has save until an other fields is change,
+see write method in contract.py for comments.
+
+so you can add notes to every amendment and set notes outside of amendment system
+
+
+Installation
+============
+
+To install this module, you need to:
```

Comment:
i see this algorithm a little complex. (I mean Overloading write function to make such changes.
What about 
* set job_id, wage as readonly on hr.contract
* create a wizard to allow to change

Reply:
@feketemihai   @legalsylvain  

ok you got to choose what is the rule and apply for all  
i've write 
"just install it" 
I change one a time with standard (no really more value for the user)
Now you ask to remove  are working for the same repository ?